charts/newrelic-infrastructure-beta/templates/control-plane/clusterrolebinding.yaml

```
@@ -0,0 +1,15 @@
+{{- if and (.Values.controlPlane.enabled) (.Values.rbac.create) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
```

All the `labels` are not correct.
```suggestion
  labels:
    {{ include "newrelic.labels" . | nindent 4 }}
```